src/Plugin/Field/FieldWidget/InlineEntityFormMultiple.php

```diff
@@ -240,7 +240,6 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $element['#element_validate'] = array('inline_entity_form_update_row_weights');
     // Add the required element marker & validation.
     if ($element['#required']) {
-      $element['#title'] .= ' ' . _theme('form_required_marker', array('element' => $element));
```

Comment:
Regarding issue, required display is now handled by css. theme_form_required_marker() doesn't exist anymore.

In my local env, after deleting this, i always have * near title thanks to $element['#required']. So i think this line can be deleted . In any case, this line break php.